config/webpack/getThemeWebpackPromise.js

```diff
@@ -0,0 +1,120 @@
+const webpack = require('webpack');
+const MemoryFS = require('memory-fs');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const rtl = require('postcss-rtl');
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const Logger = require('../../scripts/utils/logger');
+
+/**
+ * This is a basic configuration to webpack just the theme file and grab the resulting css.  Thus,
+ * there are no JS, raw, etc. loaders. The resulting themeable variables and values will be populated in
+ * the cachedObject
+ * @param {String} rootPath the root path from where the webpack is being run
```

Comment:
I think it'd be worth mention this does not support hot reloading and then will only compile the default theme defined int he theme configuration.

Reply:
https://github.com/cerner/terra-toolkit/pull/322/commits/3555c6105d7c7120831210210fde31f130e4c0d8